src/components/Card/Card.tsx

```diff
@@ -24,14 +28,18 @@ export const Card: React.FC<Props> = ({
   className,
   children,
   inverted,
+  href,
   ...other
 }) => {
   const componentClassName = clsx(styles['card'], className, {
     [styles['card--inverted']]: inverted,
   });
+
+  const TagName = href ? 'a' : 'div';
```

Comment:
Should we be concerned at all about `Card`s having interactive elements inside `children`? Also a link probably shouldn't have that much text for screen readers.

The way we're handling this in `traject` is we have an `onClick` on the card that correlates to some link in the card's contents (usually the header text). https://github.com/FB-PLP/traject/blob/80fa0f387e3c510dd4d070b1210269a1dd74297f/app/assets/javascripts/v2/core/EDSCandidates/Card/Card.tsx#L74 This is from the "redundant click solution" in the [inclusive components Card post](https://inclusive-components.design/cards/).